AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -48,10 +48,33 @@ define([
                 }
             ],
             listens: {
-                '${ $.sortByComponentName }:applied': 'hide'
+                '${ $.sortByComponentName }:applied': 'hide',
+                'adobe_stock_images_listing.adobe_stock_images_listing.listing_top.bookmarks:activeIndex': 'onActiveIndexChange'
```

Comment:
Please add the bookmarksProvider to the defaults to be able to configure it for adobe stock listing and standalone adobe stock listing separately:
 - `AdobeStockImageAdminUi/view/adminhtml/ui_component/standalone_adobe_stock_images_listing.xml`
 - `AdobeStockImageAdminUi/view/adminhtml/ui_component/adobe_stock_images_listing.xml`